proxysql/dockerdir/entrypoint.sh

```diff
@@ -3,12 +3,23 @@
 set -o xtrace
 
 PROXY_CFG=/etc/proxysql/proxysql.cnf
-PROXY_ADMIN_CFG=/etc/proxysql-admin.cnf
+# Percona scheduler
+PERCONA_SCHEDULER_CFG=/etc/config.toml
+temp_proxy_scheduler_cfg=$(mktemp)
+# proxysql-admin scheduler
+PROXY_SCHEDULER_CFG=/etc/proxysql-admin.cnf
 
 if [ -n ${PROXYSQL_SERVICE} ]; then
     MYSQL_INTERFACES='0.0.0.0:3306;0.0.0.0:33062'
     CLUSTER_PORT='33062'
-    sed "s/#export WRITERS_ARE_READERS=.*$/export WRITERS_ARE_READERS='yes'/g" ${PROXY_ADMIN_CFG} 1<> ${PROXY_ADMIN_CFG}
+	# Percona scheduler
+	sed "s/^writerIsAlsoReader.*=.*$/writerIsAlsoReader = 1/" ${PERCONA_SCHEDULER_CFG} | \
+	sed "s/^hgW.*=.*$/hgW = 11/" | \
+	sed "s/^hgR.*=.*$/hgR = 10/" | \
+	sed "s/^clustered.*=.*false$/clustered = true/" > ${temp_proxy_scheduler_cfg}
+	cp ${temp_proxy_scheduler_cfg} ${PERCONA_SCHEDULER_CFG}
```

Comment:
maybe we need to add '-f'

Reply:
Thanks! Fixed: fa1754abdbfffbea1c364d1cd33689288d881aea